<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/strelochki.png</normaloff>:/new/prefix1/strelochki.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>60</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equally">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 0 px;
	
	
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.326829 rgba(255, 255, 255, 0), stop:0.339806 rgba(255, 139, 24, 255), stop:0.668293 rgba(255, 139, 24, 255), stop:0.678049 rgba(255, 255, 255, 0));
}
QPushButton:pressed{
	
	background-color: rgba(255, 255, 255, 0);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/from.png</normaloff>:/new/prefix1/from.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="statusTip">
     <string>+</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="statusTip">
     <string>-</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>3</number>
    </property>
    <property name="statusTip">
     <string>*</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/krest.png</normaloff>:/new/prefix1/krest.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backspace">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/backspace.png</normaloff>:/new/prefix1/backspace.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>54</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>4</number>
    </property>
    <property name="statusTip">
     <string>/</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image1.qrc">
      <normaloff>:/new/prefix1/razdel.png</normaloff>:/new/prefix1/razdel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
	color: rgb(255, 139, 24);
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
	color: rgb(255, 139, 24);
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>246</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
	color: rgb(255, 139, 24);
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_round">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
	color: rgb(255, 139, 24);
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px solid gray;
	color: rgb(255, 139, 24);
}
QPushButton:pressed{
background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.532, fx:0.5, fy:0.5, stop:0.102439 rgba(238, 238, 238, 156), stop:0.395122 rgba(0, 0, 0, 6), stop:1 rgba(255, 255, 255, 11))
}
</string>
    </property>
    <property name="text">
     <string>&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>246</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_divide_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image1.qrc">:/new/prefix1/razdel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_multiply_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image1.qrc">:/new/prefix1/krest.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_plus_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image1.qrc">:/new/prefix1/plus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_minus_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image1.qrc">:/new/prefix1/minus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>171</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Калькулятор градусов может точно производить простейшие операции над градусами, при соблюдении этих правил:&lt;/p&gt;&lt;p&gt;1) Умножать можно только градусы на число (не наоборот).&lt;/p&gt;&lt;p&gt;2) Делить можно как градусы на число, так и градусы на градусы.&lt;/p&gt;&lt;p&gt;3) Остальные действия можно производить только с градусами&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image1.qrc"/>
 </resources>
 <connections/>
</ui>
